msteams-platform/concepts/deploy-and-publish/appsource/prepare/teams-store-validation-guidelines.md

```diff
@@ -474,6 +474,7 @@ Your icons must communicate your app's brand and purpose while adhering to the f
    :::image type="content" source="../../../../assets/images/submission/color-outline-icon-not-same.png" alt-text="Screenshot shows color icon and outline icon aren’t same.":::
 
 * Your app package must include two .png versions of your app icon: A color icon and an outline icon. [*Mandatory Fix*]
+* Your app icon in the Teams Store must match the app icon indicated in your app manifest. [*Mandatory Fix*]
```

Comment:
Your app icon displayed in the Teams Store must match the app icon specified in your app manifest. [*Mandatory Fix*]

Reply:
Updated